### ISSUE TITLE
Make a logo for `Mirage UI`

### ISSUE THREAD

**Issue by Pranav016** [ALPHAVIO/Mirage-UI#8]
### Feature request-
Make a logo for `Mirage UI`.  Using the text only `Mirage UI`, make a very small logo with of styled wordings. I think rainbow style would look good but you can make multiple styles and we can chose between them.

Also I saw a logo the other day that was rainbow text and had some animation to it (probably a GIF), something of that sort would also look good. 

Maybe a small logo along with the styled wordings might also look good.
Sample logo of material UI- 
![image](https://user-images.githubusercontent.com/54665036/120903456-d7d42a80-c663-11eb-8971-30f0a9a4d6d9.png)

I'd advice to take this issue only if you are creative and good with designing and can make a few samples before we can decide one.

<hr/>

**Note:**

-   If you want to work on an issue, you should check if it has already been assigned to anyone. **If the issue is free** you can comment `/assign` to get the issue assigned to you.
-   If you are raising a new issue and want to work on it then also you should comment `/assign` under the issue to get it auto assigned.
- It is adviced to take up/ work on only 2 issues at the same time.
-   Please **refrain from** adding labels to your issue/pull-request on your own. It is the job of the Project Admin and the Mentors to review your issue/pull-request and add labels accordingly.

**Comment by honeybhardwaj** [comment]
![M](https://user-images.githubusercontent.com/51120790/121657881-72969400-cabe-11eb-8762-22dd2ac91cad.png)

**Comment by Pranav016** [comment]
Doesn't look good

**Comment by starrohan999** [comment]
hey, @Pranav016  sir I am interested. can I also give a demo of the logo?

**Comment by starrohan999** [comment]
![mirage](https://user-images.githubusercontent.com/71714344/121727838-0e99bd00-cb0a-11eb-8559-d743b0131854.png)
![mirage (1)](https://user-images.githubusercontent.com/71714344/121727841-0fcaea00-cb0a-11eb-81bd-7a4332ed7091.png)

**Comment by Pranav016** [comment]
Could be better.
I'd suggest make something with the text only, don't worry about the symbol in the logo for now. Try styling the text with some flashy colors and clean corners, no white backgrounds.

I'd show a reference if I could fine it again.

**Comment by starrohan999** [comment]
sure @Pranav016  sir

**Comment by Pranav016** [comment]
Any updates on this issue?

**Comment by Manasa2811** [comment]
Hey @Pranav016 , I have a design in my mind. Could I give it a try after the currently assigned person is done?

**Comment by Pranav016** [comment]
> Hey @Pranav016 , I have a design in my mind. Could I give it a try after the currently assigned person is done?

Yes, you can also try.

**Comment by Manasa2811** [comment]
https://user-images.githubusercontent.com/57994923/122447410-55335f80-cfc1-11eb-9157-9a411243b261.mp4


https://user-images.githubusercontent.com/57994923/122447453-63817b80-cfc1-11eb-882e-53a022d3203f.mp4


https://user-images.githubusercontent.com/57994923/122448451-6466dd00-cfc2-11eb-8772-c1ef3f0398eb.mp4

**Comment by Manasa2811** [comment]
![MIRAGE](https://user-images.githubusercontent.com/57994923/122451778-279ce500-cfc6-11eb-9c3f-47b27082c521.gif)
![Mirage ui 2](https://user-images.githubusercontent.com/57994923/122451792-2a97d580-cfc6-11eb-9673-961ae6aee1eb.gif)
![Mirage UI](https://user-images.githubusercontent.com/57994923/122451808-2d92c600-cfc6-11eb-9e16-79ddcfad5a2e.gif)

**Comment by starrohan999** [comment]
![mirage](https://user-images.githubusercontent.com/71714344/122457447-48683900-cfcc-11eb-9231-07b31851602a.png)

**Comment by Pranav016** [comment]
![MIRAGE](https://user-images.githubusercontent.com/57994923/122451778-279ce500-cfc6-11eb-9c3f-47b27082c521.gif)

This one looks really nice, if you could update the animation on this we can accept it.
Also. this has a black background it might not look good if someone is going through the README in light mode, any solution to that ?

**Comment by Manasa2811** [comment]
Maybe I could make the background transparent, would that work? @Pranav016

**Comment by Manasa2811** [comment]
And what about the animation? Would you like me to change it? I didn't understand

**Comment by Pranav016** [comment]
> Maybe I could make the background transparent, would that work? @Pranav016

Yes that would be great

**Comment by Pranav016** [comment]
> And what about the animation? Would you like me to change it? I didn't understand

The animation of how the text slides from the left, if you could add some other animation like a fade out etc that would be great for options but otherwise also animation looks fine 👍

**Comment by Manasa2811** [comment]
https://user-images.githubusercontent.com/57994923/122535252-00372e00-d041-11eb-9bec-9a5768f37621.mp4


https://user-images.githubusercontent.com/57994923/122535273-04634b80-d041-11eb-81bb-ed912a83a527.mp4

**Comment by Manasa2811** [comment]
![mirage (2)](https://user-images.githubusercontent.com/57994923/122535675-69b73c80-d041-11eb-88ef-aee0cff68e02.gif)
![mirage (1)](https://user-images.githubusercontent.com/57994923/122535716-720f7780-d041-11eb-8f0e-19eceafee040.gif)

**Comment by Pranav016** [comment]
> ![MIRAGE](https://user-images.githubusercontent.com/57994923/122451778-279ce500-cfc6-11eb-9c3f-47b27082c521.gif)


This one looks the best out of the rest, if you could make the background transparent then that would be great.

**Comment by Manasa2811** [comment]
Ok so I have added 2 more animations. But I am not able to change the background to transparent as I don't have the paid version of the app I am using. If u want me to make another copy of this with a white background so that you could use both I could do that, but I'm not able to make it transparent, sorry about that.

**Comment by Pranav016** [comment]
> Ok so I have added 2 more animations. But I am not able to change the background to transparent as I don't have the paid version of the app I am using. If u want me to make another copy of this with a white background so that you could use both I could do that, but I'm not able to make it transparent, sorry about that.

Ok, you can make the PR adding this logo to the README👍

**Comment by Manasa2811** [comment]
okay, sure will do it.

**Comment by aanchalrakheja** [comment]
would like to work on this issue

**Comment by aanchalrakheja** [comment]
![mui logo](https://user-images.githubusercontent.com/68388581/124580522-d7e76600-de6d-11eb-965e-454d37c46e07.png)

**Comment by Pranav016** [comment]
Could be better

**Comment by aanchalrakheja** [comment]
![mui logo2](https://user-images.githubusercontent.com/68388581/124590154-72e53d80-de78-11eb-8ad2-e6101cca6604.png)
![mui logo3](https://user-images.githubusercontent.com/68388581/124593345-2865c000-de7c-11eb-9da1-b4859317b494.png)
Something like this ?

**Comment by aanchalrakheja** [comment]
![mui logo 4](https://user-images.githubusercontent.com/68388581/124598178-f7888980-de81-11eb-8f23-b84fe31a48d7.png)

**Comment by Pranav016** [comment]
![image](https://user-images.githubusercontent.com/54665036/125079778-a1b21c80-e0e1-11eb-9923-65fbf989cdba.png)
 First one looks good, for now add it to the assets folder of the project. I'll manage the rest 👍

**Comment by aanchalrakheja** [comment]
Ok sure
